client/styles/components/_sidebar.scss

```diff
@@ -17,6 +16,7 @@
 .sidebar__title {
   font-size: #{12 / $base-font-size}rem;
   display: inline-block;
+  color: #a8a8a8;
```

Comment:
could you put this color into a variable and add it to the themes so it looks good in the light dark, and high contrast themes?